### DISCUSSION TITLE
Plugin main translations - Are there set in English for any specific reason?

### DISCUSSION THREAD

**Discussion by xmontero** [benkeen/generatedata#773] [General]
I see that in the plugins, you set translations "about a country" inside the `i18n` dir.

For example the file

    src/plugins/countries/SouthKorea/i18n/en.json

contains:

    {
      "countryName": "South Korea",
      "regionNames": "South Korea provinces"
    }

Nevertheless, the same directory, for the file `es.json` it contains a clone of the English version instead of a Spanish version that should be:

    {
      "countryName": "Corea del Sur",
      "regionNames": "Provincias de Corea del Sur"
    }

Should I translate that into Spanish? Or they are "forced" into English for any special reason?

**Comment by benkeen** [comment]
Good catch: bug! Yes, it should be translated - not English. I'll create a ticket for it - I see others in that plugin don't have the translations made.

**Reply by xmontero** [reply]
I've a question... in English it starts with capital as it starts with the country name, like `South Korea provinces`.

In Spanish the order is reversed with respect to English so we start with `Provincias de Corea del Sur`. Should it be capitalized? Or should it start in lowercase like `provincias de Corea del Sur`?

What's the context this text is used in?

**Reply by benkeen** [reply]
Hmm, this actually appears to be a holdover from a previous version where that information was displayed directly in the UI. But with the larger number of countries that we now have, the UI was simplified and this string hidden. Still, translate it anyway - I'll create a ticket to examine it and trim it out if it's no longer user. 

I'd stick with starting with uppercase.

**Reply by andrefortin** [reply]
I believe that "regionNames" was originally intended to answer the question: "What does this country call its regional subdivisions?"

For example in Canada it would be, in English, "Provinces". (We do have Territories as well so it could reasonably be "Provinces and Territories"
{
  "countryName": "Canada",
  "regionNames": "Provinces"
}

In the USA it would be, in English, "States".
{
  "countryName": "United States",
  "regionNames": "States"
}

Therefore, in your example for South Korea, it would be
{
  "countryName": "Corea del Sur",
  "regionNames": "Provincias"
}
